### PR TITLE
Update 6-2 to follow BASHER

### PR DESCRIPTION
**Pull request by derisen**
## Purpose

* Update project dependencies
* Update README
* Update registration scripts

## Does this introduce a breaking change?

```
[ ] Yes
[ x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ x ] Code style update (formatting, local variables)
[ x ] Refactoring (no functional changes, no api changes)
[ x ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: 6-AdvancedScenarios/2-call-api-mt/README.md [hunk 1]
```diff
@@ -84,117 +105,177 @@ For more information and potential issues, see: [HTTPS in .NET Core](https://doc
     npm install
```
Comment (salman90) [inline 6-AdvancedScenarios/2-call-api-mt/README.md:105]: Add the path for the SPA

File: 6-AdvancedScenarios/2-call-api-mt/SPA/src/app/todo-view/todo-view.component.ts [hunk 2]
```diff
@@ -1,64 +1,69 @@
...
+                console.log(todos)
+                this.todos = todos;
+            });
+    }
 
-  checkTodo(todo: Todo): void {
-    this.service.editTodo(todo).subscribe();
-  }
+    addTodo(add: NgForm): void {
+        console.log(add.value)
```
Comment (salman90) [inline 6-AdvancedScenarios/2-call-api-mt/SPA/src/app/todo-view/todo-view.component.ts:53]:
```suggestion
```

File: 6-AdvancedScenarios/2-call-api-mt/SPA/src/app/todo-view/todo-view.component.ts [hunk 3]
```diff
@@ -1,64 +1,69 @@
...
-  addTodo(add: NgForm): void {
-    this.service.postTodo(add.value).subscribe(() => {
-      this.getTodos();
-    })
-    add.resetForm();
-  }
+    getTodos(): void {
+        this.service.getTodos()
+            .subscribe((todos: Todo[]) => {
+                console.log(todos)
```
Comment (salman90) [inline 6-AdvancedScenarios/2-call-api-mt/SPA/src/app/todo-view/todo-view.component.ts:47]:
```suggestion
```

**Comment by v-michaelmi** [comment]
Couple of nits.

Otherwise LGTM 👍

File: 6-AdvancedScenarios/2-call-api-mt/SPA/src/app/claim-utils.ts [hunk 4]
```diff
@@ -0,0 +1,207 @@
+/**
+ * Populate claims table with appropriate description
+ * @param {Record} claims ID token claims
+ * @returns claimsTable
+ */
+export const createClaimsTable = (claims: Record<string, string>): any[] => {
+    const claimsTable: any[] = [];
+
+    Object.keys(claims).map((key) => {
```
Comment (salman90) [inline 6-AdvancedScenarios/2-call-api-mt/SPA/src/app/claim-utils.ts:9]: Update method to check for types, [see](https://github.com/Azure-Samples/ms-identity-javascript-react-tutorial/pull/162/files)

File: 6-AdvancedScenarios/2-call-api-mt/SPA/src/app/auth-config.ts [hunk 5]
```diff
@@ -5,26 +5,32 @@
...
+ * https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/configuration.md
  */
 export const msalConfig: Configuration = {
     auth: {
         clientId: 'Enter_the_Application_Id_Here', // This is the ONLY mandatory field that you need to supply.
         authority: 'https://login.microsoftonline.com/organizations', // Defaults to "https://login.microsoftonline.com/common"
-        redirectUri: '/', // Points to window.location.origin. You must register this URI on Azure portal/App Registration.
+        redirectUri: '/auth', // Points to window.location.origin by default. You must register this URI on Azure portal/App Registration.
+        postLogoutRedirectUri: '/', // Points to window.location.origin by default.
+        clientCapabilities: ['CP1'] // This lets the resource server know that this client can handle claim challenges.
```
Comment (salman90) [inline 6-AdvancedScenarios/2-call-api-mt/SPA/src/app/auth-config.ts:23]: Do we need clientCapabilities?

**Review by salman90 — APPROVED** [review]
Looks good!

**Review by kalyankrishna1 — APPROVED** [review]
:shipit:

File: 6-AdvancedScenarios/2-call-api-mt/SPA/src/app/todo-view/todo-view.component.ts (hunk 2 above)
Comment (derisen) [reply 6-AdvancedScenarios/2-call-api-mt/SPA/src/app/todo-view/todo-view.component.ts:52]: removed

File: 6-AdvancedScenarios/2-call-api-mt/SPA/src/app/auth-config.ts (hunk 5 above)
Comment (derisen) [reply 6-AdvancedScenarios/2-call-api-mt/SPA/src/app/auth-config.ts:23]: good catch -removed

File: 6-AdvancedScenarios/2-call-api-mt/SPA/src/app/claim-utils.ts (hunk 4 above)
Comment (derisen) [reply 6-AdvancedScenarios/2-call-api-mt/SPA/src/app/claim-utils.ts:9]: done

**Pull request merged**